cypress/e2e/scenario-4-desired-state.cy.js

```diff
@@ -554,18 +518,12 @@ describe("Scenario 4 Desired State", () => {
       }
     });
     // expect the view to still contain the diff of the last dry-run comparison
-    cy.get(".pf-v5-c-card__expandable-content", { timeout: 20000 }).should(
+    cy.get(".pf-v6-c-card__expandable-content", { timeout: 20000 }).should(
```

Comment:
Is this correct for iso7?

Reply:
Right, I missed that and was confused for a moment why the test wasn't passing 😅